src/main/scala/com/datastax/gatling/plugin/request/CqlRequestAction.scala

```diff
@@ -61,18 +61,10 @@ class CqlRequestAction(val name: String,
   }
 
   def sendQuery(session: Session): Unit = {
-    val enableCO = Boolean.getBoolean("gatling.dse.plugin.measure_service_time")
-    val responseTimeBuilder: ResponseTimeBuilder = if (enableCO) {
-      // The throughput checker is useless in CO affected scenarios since throughput is not known in advance
-      COAffectedResponseTime.startingAt(System.nanoTime())
-    } else {
-      ThroughputVerifier.checkForGatlingOverloading(session, gatlingTimingSource)
-      GatlingResponseTime.startedByGatling(session, gatlingTimingSource)
-    }
     val stmt = dseAttributes.statement.buildFromSession(session)
 
     stmt.onFailure(err => {
-      val responseTime: ResponseTime = responseTimeBuilder.build()
+      val responseTime = new ServiceTime(0, 0)
```

Comment:
Would it be clearer to name this `serviceTime` instead of `responseTime`?

This would also require a change to line 71:

```
statsEngine.logResponse(session, name, serviceTime.toGatlingResponseTimings, KO, None,
         Some(s"$tagString - Preparing: ${err.take(50)}"), List(responseTime.latencyIn(MICROSECONDS), "PRE", logUuid))		         Some(s"$tagString - Preparing: ${err.take(50)}")
```

but to _me_ that seems a bit clearer. We are converting a `ServiceTime` to a `ResponseTime`. Thoughts?